examples/_tests_mnist_stages/__init__.py

```diff
@@ -1,4 +1,6 @@
 # flake8: noqa
-from catalyst.dl import SupervisedRunner as Runner
+from catalyst.dl import registry, SupervisedRunner as Runner
 from .experiment import Experiment
 from .model import SimpleNet
+
+registry.Model(SimpleNet)
```

Comment:
why do wee need this, if we have already registred `SimpleNet`?
https://github.com/catalyst-team/catalyst/blob/master/examples/_tests_mnist_stages/model.py#L7